### PR TITLE
Adding documentation for BigInt.to_s methods

### PR DESCRIPTION
**Pull request by Nesqwik**
In src/big_int/big_int.cr :
- add documentation for to_s

### CODE REVIEW

File: src/big_int/big_int.cr [hunk 1]
```diff
@@ -202,15 +202,29 @@ struct BigInt < Int
     to_s io
   end
 
+  # Returns a string representation of self.
+  #
+  # ```
+  # puts 12345654321.to_s # => 12345654321
```
Comment (asterite) [inline src/big_int/big_int.cr]: `12345654321` is not a BigInt. Should be something more like:

```crystal
puts BigInt.new("1234565432112345654321").to_s # => 1234565432112345654321
```

**Comment by Nesqwik** [comment]
Thank's for the comment. It should be a BigInt now !

**Comment by asterite** [comment]
@Nesqwik Thank you! Could you squash into a single commit?

**Comment by Nesqwik** [comment]
It's done now, sorry for taking so long.

**Comment by jhass** [comment]
Thanks!

**Pull request merged**